src/solvers/dgmulti/dg.jl

```diff
@@ -104,540 +121,563 @@ Return an iterator over the indices that specify the location in relevant data s
 for the face nodes in `mesh`.
 In particular, not the face nodes themselves are returned.
 """
-@inline each_face_node_global(mesh::DGMultiMesh, dg::DGMulti, other_args...) = Base.OneTo(dg.basis.Nfq * mesh.md.num_elements)
+@inline function each_face_node_global(mesh::DGMultiMesh, dg::DGMulti, other_args...)
+    Base.OneTo(dg.basis.Nfq * mesh.md.num_elements)
+end
 
 # interface with semidiscretization_hyperbolic
 wrap_array(u_ode, mesh::DGMultiMesh, equations, dg::DGMulti, cache) = u_ode
 wrap_array_native(u_ode, mesh::DGMultiMesh, equations, dg::DGMulti, cache) = u_ode
-function digest_boundary_conditions(boundary_conditions::NamedTuple{Keys, ValueTypes}, mesh::DGMultiMesh,
-                                    dg::DGMulti, cache) where {Keys, ValueTypes<:NTuple{N, Any}} where {N}
-  return boundary_conditions
+function digest_boundary_conditions(boundary_conditions::NamedTuple{Keys, ValueTypes},
+                                    mesh::DGMultiMesh,
+                                    dg::DGMulti,
+                                    cache) where {Keys, ValueTypes <: NTuple{N, Any}
+                                                  } where {N}
```

Comment:
Not sure how I feel about the formatting inside the curly braces in the `where ...` statement